styleguide/components/41-organisms/teaser-collections/teaser-topic-collection/teaser-topic-collection.twig

```diff
@@ -0,0 +1,16 @@
+<div class="collection teaser-topic-collection">
+  <h2>{{ 'Topics' }}</h2>
```

Comment:
Suggest adding a span to this title and placing the accolade (?) on the span so it has the same with as the title, whilst retaining the full width blue line.
That way it's also possible to only add the accolade to the first word, should this be needed in future.

Reply:
@delrueba I'll check with design. They should decide what the behaviour of this accolade should be.